server/src/controllers/callback.controller.ts

```diff
@@ -0,0 +1,91 @@
+import { Request, Response, Router } from "express";
+import { MongoClient } from "mongodb";
+import { DI } from "../server";
+import qs from 'qs';
+async function createUser(user: any): Promise<any> {
```

Comment:
We are already connected to mongodb locally, why are we connecting to another cloud database?